### ISSUE TITLE
NTR: N-carbamyl-beta-aminoisobutyric acid high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8293]
New term request N-carbamyl-beta-aminoisobutyric acid high in blood (elthat_111121052026):

**Comment by pnrobinson** [comment]
@MickeySegal
added as
HP:0034595
Elevated circulating N-carbamyl-beta-alanine concentration

**Comment by MickeySegal** [comment]
Alanine (https://en.wikipedia.org/wiki/Alanine) and aminobutyric acid (https://en.wikipedia.org/wiki/%CE%92-Aminobutyric_acid) are different compounds.  Is this for aminobutyric as in this new term request or for alanine as in  https://github.com/obophenotype/human-phenotype-ontology/issues/8294 ?

**Comment by pnrobinson** [comment]
@MickeySegal 
sorry, I pasted the wrong new term I will reopen and add N-carbamyl-beta-aminoisobutyric acid  as well

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000696
- label: Elevated circulating N-carbamyl-beta-aminoisobutyric acid concentration
- parents: [Abnormal circulating monocarboxylic acid concentration (HP:0010996)](https://hpo.jax.org/app/browse/term/HP:0010996)
- pmid: PMID:15385443
- def: The concentration of N-carbamyl-beta-aminoisobutyric acid in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8293
- synonyms:
